README.md

```diff
@@ -117,7 +112,8 @@ $ rails new . -d postgresql
 ```
 
 ```
-Your answer...
+The first command creates a new rails file called Tunr.
+The second command connects a new file to a PostgreSQL database
```

Comment:
The first creates a new rails app called `tunr` while the second while the second installs all rails files inside the present working directory, specifically with a postgresql db